File: App2/src/app/_core/_model/customer.ts
export interface Customer {
  id: number;
  customerNo: string;
  customerName: string;
  customerSex: string;
  customerSexName: string;
  customerBirthday: string | null;
  customerNickname: string;
  customerTel: string;
  customerMobile: string;
  customerAddress: string;
  customerIdcard: string;
  customerEmail: string;
  contactName: string;
  contactTel: string;
  contactMobile: string;
  contactEmail: string;
  comment: string;
  cancelFlag: string;
  createDate: string | null;
  createBy: number | null;
  updateDate: string | null;
  updateBy: number | null;
  status: number | null;
  guid: string;
  farmGuid: string;
}

File: PigFarm/Models/Customer.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PigFarm.Models
{
    public partial class Customer
    {
        public decimal Id { get; set; }
        public string CustomerNo { get; set; }
        public string CustomerName { get; set; }
        public string CustomerSex { get; set; }
        public DateTime? CustomerBirthday { get; set; }
        public string CustomerNickname { get; set; }
        public string CustomerTel { get; set; }
        public string CustomerMobile { get; set; }
        public string CustomerAddress { get; set; }
        public string CustomerIdcard { get; set; }
        public string CustomerEmail { get; set; }
        public string ContactName { get; set; }
        public string ContactTel { get; set; }
        public string ContactMobile { get; set; }
        public string ContactEmail { get; set; }
        public string Comment { get; set; }
        public string CancelFlag { get; set; }
        public DateTime? CreateDate { get; set; }
        public decimal? CreateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public decimal? UpdateBy { get; set; }
        public decimal? Status { get; set; }
        public string Guid { get; set; }
        public string FarmGuid { get; set; }
    }
}
